core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -69,13 +73,13 @@ private[ui] class ExecutorsPage(
     }
 
     val execInfo = activeExecutorInfo ++ deadExecutorInfo
-    val execInfoSorted = execInfo.sortBy(_.id)
+    val execInfoSorted = execInfo.sortWith((a, b) => strToInt(a.id) > strToInt(b.id))
```

Comment:
Thanks, I've switched to using `Ordering` but decided to leave the Try in the function since I also use it in the table sorting. I'll push the update once I test it.